components/service/glean/src/main/java/mozilla/components/service/glean/scheduler/BaselinePingScheduler.kt

```diff
@@ -0,0 +1,135 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.service.glean.scheduler
+
+import android.view.accessibility.AccessibilityManager
+import android.accessibilityservice.AccessibilityServiceInfo
+import android.content.Context
+import android.content.SharedPreferences
+import android.os.Build
+import mozilla.components.service.glean.metrics.GleanBaseline
+import mozilla.components.support.base.log.logger.Logger
+import java.util.concurrent.TimeUnit
+
+/**
+ * BaselinePingScheduler facilitates sending and scheduling of baseline pings when the application
+ * enters the background state (see [GleanLifecycleObserver]).  Using a [PING_INTERVAL_HOURS] of
```

Comment:
`PING_INTERNAL_HOURS` is 0, so it's basically not doing anything useful. Let's remove it and update this description to the current ping behaviour: we're sending the ping every time we go to background.